Paraswap-Agents/delegate-calls/src/agent.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  TransactionEvent,
+  HandleTransaction,
+  getEthersProvider,
+} from "forta-agent";
+
+import { ethers } from "ethers";
```

Comment:
`FindingSeverity`, `FindingType`, `getEthersProvider` and `ethers` are not used in this file.